src/Sulu/Bundle/AudienceTargetingBundle/Content/Types/AudienceTargetingGroups.php

```diff
@@ -43,7 +43,7 @@ public function __construct(TargetGroupRepositoryInterface $targetGroupRepositor
      */
     public function getType()
     {
-        return ContentTypeInterface::PRE_SAVE;
+        return ContentEvents::NODE_PRE_SAVE;
```

Comment:
This is not the right usage of `ContentEvents`... Actually the `getType` method was banned from the `ContentTypeInterface` completely, so we should probably just delete this method.